integration-tests/tests/multiparty.spec.ts

```diff
@@ -133,6 +133,7 @@ test.describe('display name for screenshare', () => {
     });
 
     await pageOne.waitForSelector('.publisher', { state: 'visible' });
+    await pageTwo.waitForSelector('.subscriber', { state: 'visible' });
     const screenshareButton = await pageOne.getByTestId('ScreenShareIcon');
```

Comment:
I think this was failing sometimes because we were connected so the screenshare button wasn't available yet.
For now this fix waits until a subscriber is visible for tab 2 which means we're obviously already connected and publishing & subscribing - then starts screensharing from page one